src/bindings/config.ts

```diff
@@ -60,7 +60,7 @@ export const loadConfig = async (context: Context): Promise<BotConfig> => {
     payout: {
       networkId: networkId,
       rpc: rpc,
-      privateKey: privateKey,
+      privateKeyEncrypted: privateKeyEncrypted || "",
```

Comment:
You should default to `null` instead of an empty string. It makes the code more readable.  

```
if (privateKeyEncrypted)
```

It also clearly expresses that it is intentionally set to nothing vs some type of bug. 

Otherwise the code looks good.

Reply:
Fixed https://github.com/ubiquity/ubiquibot/pull/858/commits/9c587460af1b92cadaa2d50ca524e3c1c652dc14